cpp/src/io/orc/aggregate_orc_metadata.cpp

```diff
@@ -220,27 +219,25 @@ aggregate_orc_metadata::select_stripes(
   }
 
   // Read each stripe's stripefooter metadata
-  if (not selected_stripes_mapping.empty()) {
```

Comment:
How do we know if it's safe to remove this `if`? (I am assuming the other changes in this section are from the change in indentation. If I've missed logical changes, please point them out.)

Reply:
Ah. I thought there was more to consider when I quickly reviewed this the first time. I agree this is fine.